src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
+type Color = 'red' | 'green' | 'blue';
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
```

Comment:
It is inconsistent to use enums and unions for the same thing together
Please pick one of them